nav2_util/include/nav2_util/is_localized.hpp

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2019 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef NAV2_UTIL__IS_LOCALIZED_HPP_
+#define NAV2_UTIL__IS_LOCALIZED_HPP_
+
+#include <string>
+#include "rclcpp/rclcpp.hpp"
+#include "nav2_robot/robot.hpp"
+
+namespace nav2_util
+{
+
+class IsLocalized
+{
+
+public:
+  IsLocalized();
+  bool isLocalized();
+
+private:
+  static const int cov_x_ = 0;
```

Comment:
Can you put a comment explaining what these magic numbers mean and where they come from? Looking below, I understand they're indexes in a covariance matrix, but that's not clear here. Maybe rename them to cov_x_index_ too.